Example/Example/SomeView.swift

```diff
@@ -16,7 +17,7 @@ struct SomeView: View {
         Text("Hello from SwiftUI 🙃")
             .onAppear {
                 span.end()
```

Comment:
Can you remove the manual instrumentation, and also remove the "(manual)" from the buttons on the main screen of the example app.